docs/docs/Methodology/Converting Gross to Net Generation.md

```diff
@@ -54,12 +54,11 @@ The conversion factors are applied in the following hierarchical order:
 
 1. Subplant gross-to-net ratio
 2. Plant gross-to-net ratio
-3. Subplant shift factor
-4. Plant shift factor
-5. Fuel-specific ratio
-6. Set net generation equal to net generation
+3. Fleet-specific ratio
+4. Default GTN ratios from EIA 
+5. Set net generation equal to net generation
```

Comment:
Set net net generation equal to gross generation?

Reply:
Updated, we actually no longer use this method